client/app/bundles/HelloWorld/components/NewResturantForm.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class NewResturantForm extends React.Component {
+
+    static propTypes = {
+        cuisines: PropTypes.array.isRequired
+    };
+
+    /*componentDidMount() {
+        console.log('mounted');
+        navigator.geolocation.getCurrentPosition((position) => {
+            console.log(position)
+        })
+    }*/
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: '',
+            cuisine: '',
+            tenbis: false,
+            address: '',
+            maxDeliveryTime: ''
+        }
+    };
+
+    newResturantSubmit = (e) => {
+        e.preventDefault();
+        $.ajax({
+            url: '/resturants',
+            dataType: 'json',
+            type: 'POST',
+            data: {resturant: {name: this.state.name,
+                tenbis: this.state.tenbis,
+                address: this.state.address,
+                max_delivery_time: this.state.maxDeliveryTime,
+                cuisine_id: this.state.cuisine},
+            },
+            success: function() {
+                window.location.href = "/";
+            }
+        });
+    };
+
+    handleNameChange = (e) => {
+        this.setState({
+            name: e.target.value
+        });
+    };
+
+    handleCuisineChange = (e) => {
+        this.setState({
+            cuisine: parseInt(e.target.value)
```

Comment:
You can use the `+e.target.value` operator as a shorthand or `Number(e.target.value)` in ES6. In any case, if you use `parseInt` you should always pass the base https://coderwall.com/p/4eaixa/parseint-can-be-dangerous